chatterino.pro

```diff
@@ -583,6 +585,7 @@ HEADERS += \
     src/widgets/settingspages/IgnoresPage.hpp \
     src/widgets/settingspages/KeyboardSettingsPage.hpp \
     src/widgets/settingspages/ModerationPage.hpp \
+    src/widgets/settingspages/NicknamesPage.hpp \
```

Comment:
After creating files for `RatelimitBucket` I ran `tools/update_filelist.py` and script also added this missing header on its own, I guess it could be sneaked in as a missing part of #2981.